src/Template/Layout/js/app/app.js

```diff
@@ -298,41 +319,54 @@ const _vueInstance = new Vue({
             }
         },
 
+
         /**
          * build coherent url based on these params:
-         * - q= query string
-         * - page_size
+         * - q=_string_
+         * - filter[_name_]=_value_
+         * - page_size=_value_
          *
          * @param {Object} params
          * @returns {String} url
          */
         buildUrlParams(params) {
             let url = `${window.location.origin}${window.location.pathname}`;
-            let first = true;
-            let queryId = '?';
+            const queryId = '?';
             const separator = '&';
-
-            Object.keys(params).forEach((key) =>  {
-                if (params[key] && params[key] !== '') {
-                    const query = params[key];
-                    let entry = `${key}=${query}`;
-
-                    // parse filter property
-                    if (key === 'filter') {
-                        let filter = '';
-                        Object.keys(query).forEach((filterKey) => {
-                            if (query[filterKey] !== '') {
-                                filter += `filter[${filterKey}]=${query[filterKey]}`;
-                            }
-                        });
-
-                        entry = filter;
+            const paramsKeys = Object.keys(params);
+
+            if (paramsKeys && paramsKeys.length) {
+                let fields = [];
+
+                paramsKeys.forEach((key) =>  {
+                    if (params[key] && params[key] !== '') {
+                        const query = params[key];
+
+                        // parse filter property
+                        if (key === 'filter') {
+                            let filter = '';
+                            Object.keys(query).forEach((filterKey) => {
+                                if (typeof query[filterKey] === 'object') {
+                                    const filter = query[filterKey];
+                                    Object.keys(filter).forEach((modifier) => {
+                                        if (filter[modifier] !== '') {
+                                            // cerca modificatori
```

Comment:
you mean: `find modifiers` ?? :smiley_cat:

Reply:
yep